src/api/api.js

```diff
@@ -235,3 +235,7 @@ export function getAllTags(search, { filters, pagination, ...options } = { pagin
         }
     );
 }
+
+export function getOperatingSystems() {
+    return fetch('/api/inventory/v1/system_profile/operating_system').then(response => response.json());
```

Comment:
Please don't use `fetch` we have an JS client for this available https://github.com/RedHatInsights/javascript-clients/blob/master/packages/host-inventory/doc/classes/systemprofileapi.md#apisystemprofilegetoperatingsystem
```suggestion
    return systemProfile.apiSystemProfileGetOperatingSystem()
```

You can (and probably should) also pass other applied filters to filter further down the list of OS. You'll probably have to import `SystemProfileApi` from inventory javascript client and add `const systemProfile = new SystemProfileApi(undefined, INVENTORY_API_BASE, instance);

Reply:
@karelhala, my first commits were containing the code that you have just suggested. However, the `systemProfile` client was introduced in later versions of `javascript-clients`, and that meant I also needed to upgrade the package in the Inventory app. However, after doing that, we ran into multiple issues caused by the breaking changes in the API, please see https://github.com/RedHatInsights/javascript-clients/issues/166. The number of bugs appeared caused by the upgrade of this client seemed to us inappropriate to solve within this PR, and we decided to take the golden path and implement the request by this one liner.

To your second comment: we are asked not to modify the list of OS versions. That means, there is no need to provide parameters to the call to `system_profile/operating_system`.

Reply:
@gkarat understandable, let's open another PR where we update the version of the client to stay updated and we can use the version here as well. It would be really bad if half of the request were done with `fetch` and the other with client itself.

Reply:
Sounds reasonable, thank you

Reply:
Actually opened a PR to use the latest JS client version https://github.com/RedHatInsights/insights-inventory-frontend/pull/1628